pkg/proto/common/v1/common.pb.go

```diff
@@ -19,9 +19,9 @@
 package common
 
 import (
+	any1 "github.com/golang/protobuf/ptypes/any"
```

Comment:
These changes look odd. Please make sure you're using the same version of the  protobuf compiler we use: https://github.com/dapr/dapr/tree/master/dapr#proto-client-generation

Reply:
Fixed.